scripts/fetch.sh

```diff
@@ -210,8 +210,9 @@ sync_references commits googleapis googleapis https://github.com/googleapis/goog
 sync_references commits googlechrome lighthouse https://github.com/GoogleChrome/lighthouse proto
 sync_references commits grpc grpc https://github.com/grpc/grpc-proto
 sync_references commits grpc-ecosystem grpc-gateway https://github.com/grpc-ecosystem/grpc-gateway
-sync_references commits opencensus opencensus https://github.com/census-instrumentation/opencensus-proto src
-sync_references commits opentelemetry opentelemetry https://github.com/open-telemetry/opentelemetry-proto
+# TODO: re-enable once unblocked
+# sync_references commits opencensus opencensus https://github.com/census-instrumentation/opencensus-proto src
```

Comment:
opencensus hasn't changed (and the repo is archived). I don't think we need to turn this off because of upstream breakage but ok if we stop syncing if there will be no more updates.

Reply:
You're right, I misread the logs, that one is not the one failing, but opentelemetry is.

I can send another PR re-enabling it, although I don't think they'll have any more updates, they "merged into" otel.

Should we just remove it altogether? wdyt @smallsamantha